cluster-autoscaler/cloudprovider/clusterapi/clusterapi_controller_test.go

```diff
@@ -76,7 +79,19 @@ func mustCreateTestController(t *testing.T, testConfigs ...*testConfig) (*machin
 
 	kubeclientSet := fakekube.NewSimpleClientset(nodeObjects...)
 	dynamicClientset := fakedynamic.NewSimpleDynamicClient(runtime.NewScheme(), machineObjects...)
-	controller, err := newMachineController(dynamicClientset, kubeclientSet)
+	discoveryClient := &fakediscovery.FakeDiscovery{
+		Fake: &clientgotesting.Fake{
+			Resources: []*v1.APIResourceList{
+				{
+					GroupVersion: "machine.openshift.io/v1beta1",
```

Comment:
Why do we reference openshift here?

Reply:
OK, I see. We're testing discovery. But, I would still choose a more agnostic name for the case that should not be found.

Reply:
this is used for validating env variable overridability, let me update it to something like "custom" to avoid confusion.

Reply:
Updated to reference `customCAPIGroup = "custom.x-k8s.io"`